ControlRoomApplication/ControlRoomApplication/Database/Operations/DatabaseOperations.cs

```diff
@@ -731,6 +731,20 @@ public static void AddSensorStatusData(SensorStatus sensors)
            }
         }
 
+        /// <summary>
+        /// Grabs the most recent sensor status data
+        /// </summary>
+        public static SensorStatus GetSensorStatusData()
```

Comment:
Is there a reason why we need this method? The sensor statuses can be read directly from the control room and database reads can be expensive.

- [ ] Consider removing `GetSensorStatusData()`